_posts/2026-1-12-op4a02.md

```
@@ -0,0 +1,107 @@
+
+### Install Docker
+
+```shell
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+```
+
+---
+
+## 2. Install ACUITY
+
+> **Note:** Due to SDK version compatibility, different Docker images are required depending on the target NPU (A733 or T527)[I used T527-OrangePi-4A]. Download the Docker image from my [Google Drive](https://drive.google.com/file/d/1U0rBrbYAlKnZhTWbbzzpgTCG8G4EKbLY/view?usp=drive_link).
```

markdownlint-cli2 (0.22.1)

[warning] Reversed link syntax
(A733 or T527)